Makefile

```
@@ -568,13 +577,28 @@ memory-leak-dind: start-prod ## Run Memlab tests in isolated compose project (DI
 	node scripts/generateLocalization.mjs
-	MUTATION_SHARD_INDEX=$(MUTATION_SHARD_INDEX) MUTATION_SHARD_TOTAL=$(MUTATION_SHARD_TOTAL) \
+	MUTATION_SCOPE=$(MUTATION_SCOPE) \
+		MUTATION_SHARD_INDEX=$(MUTATION_SHARD_INDEX) MUTATION_SHARD_TOTAL=$(MUTATION_SHARD_TOTAL) \
 		$(STRYKER_CMD) $(STRYKER_SHARD_CONFIG)
 
-merge-mutation-reports: ## Union the per-shard mutation reports and re-enforce the exact Stryker break gate over the whole set (host; assumes deps installed)
-	MUTATION_SHARD_TOTAL=$(MUTATION_SHARD_TOTAL) $(MERGE_MUTATION_REPORTS_CMD)
+merge-mutation-reports: ## Union the per-shard mutation reports and re-enforce MUTATION_SCOPE's break gate over the whole set (host; assumes deps installed)
+	MUTATION_SCOPE=$(MUTATION_SCOPE) MUTATION_SHARD_TOTAL=$(MUTATION_SHARD_TOTAL) $(MERGE_MUTATION_REPORTS_CMD)
+
+test-mutation-changed: ## Mutate only the files this branch changes against MUTATION_BASE_REF and gate on the changed-files threshold (host; assumes deps installed)
```

checkmake (0.3.2)

[warning] Target body for "test-mutation-changed" exceeds allowed length of 5 lines (10).

(maxbodylength)